kafka/client.go

```diff
@@ -125,6 +127,31 @@ func newTLSConfig(clientCert, clientKey, caCert string) (*tls.Config, error) {
 	return config, nil
 }
 
+func MakeAdminClient(clientID string, kafkaAuthCfg *utils.KafkaAuthConfig, bootstrapServers []string) (sarama.ClusterAdmin, error) {
```

Comment:
Content of this looks very similar to the block at https://github.com/knative/eventing-contrib/pull/1667/files#diff-27083a6c5d0b81eb155bf1deda5b4231e49fe42631729db2f8c7b3b55eb57d0fR92-R114

If possible, reuse the logic?